deploy/examples/operator.yaml

```diff
@@ -616,11 +616,13 @@ spec:
             - mountPath: /etc/ceph
               name: default-config-dir
           env:
+            # RevisionHistoryLimit value  for all deployments created by rook.
+            - name: ROOK_REVISION_HISTORY_LIMIT
```

Comment:
Instead of an env var, please move it up to be part of the configmap. Note that the format of the settings in the configmap is a bit different, so for example, around line 571 you would add this:
```
ROOK_REVISION_HISTORY_LIMIT: "3"
```
You shouldn't need other changes in code to accommodate this, the operator already loads them into the same `map[string]string`.

Reply:
@travisn  wrote:
> Instead of an env var, please move it up to be part of the configmap. Note that the format of the settings in the configmap is a bit different, so for example, around line 571 you would add this:
> 
> ```
> ROOK_REVISION_HISTORY_LIMIT: "3"
> ```
> 
> You shouldn't need other changes in code to accommodate this, the operator already loads them into the same `map[string]string`.

done